faw/ci/ci_container_service.py

```diff
@@ -773,6 +773,8 @@ def _create_dockerfile_contents(development, config, config_data, build_dir, bui
     # Always install observatory component dependencies as first part of final
     # stage (to minimize rebuilds on user code changes)
     dockerfile_final.append(rf'''
+            # Make root dir for static HTTP resources (must be before all stage commands)
+            RUN mkdir /home/static
```

Comment:
Not sure if this is the best place to create this directory. The dockerfile is so complex already that I'm tempted to just make it in `main.py` if it wasn't already created by copying. But this seems saner.